dataactbroker/README.md

```diff
@@ -721,6 +721,34 @@ This route certifies the specified submission, if possible. If a submission has
 ```
 * `message` - A message indicating whether or not the action was successful. Any message other than "Success" indicates a failure.
 
+#### GET "/v1/gtas_window"
+
+This route checks if there is a gtas window currently open, and if it is returns the start and end date, else returns None
+
+##### Body (JSON)
+
+```
+{}
+```
+
+##### Body Description
+
+
+##### Response (JSON)
```

Comment:
You should write what the response is. If it IS a gtas window, it returns a JSON containing a data object with the start and end dates. If it isn't, it returns a data element containing `null`